### DISCUSSION TITLE
ltsp image / error

### DISCUSSION THREAD

**Discussion by wolfclan68** [ltsp/ltsp#216] [Q&A]
I'm getting this same error when running the command image or initrd, any ideas?

/usr/sbin/ltsp: 131: eval: Syntax error: Unterminated quoted string

**Comment by alkisg** [comment]
Can you paste all of the output?

If it happens with all ltsp commands, it might be a problem with your ltsp.conf...

**Comment by wolfclan68** [comment]
That is all the output I get. I did just add to the ltsp.conf based on Daniel's Single guest session in the wiki. Something must be off, thanks for pointing me in the right direction.
